content/en/news/releases/1.5.x/announcing-1.5/_index.md

```diff
@@ -15,4 +15,106 @@ We are pleased to announce the release of Istio 1.5!
+
+## A new model for extensibility
+
+Istio has long been the most extensible service mesh, with Mixer plugins
+allowing custom policy and telemetry support and Envoy extensions allowing data
+plane customization. In Istio 1.5 we’re announcing a new model that unifies
+Istio’s extensibility model with Envoy’s, using
+[WebAssembly](https://webassembly.org/) (Wasm). Wasm will give developers the
+ability to safely distribute and execute code in the Envoy proxy -- to integrate
+with telemetry systems, policy systems, control routing and even transform the
+body of a message. It will be more flexible and more efficient, eliminating the
+need for running a Mixer component separately (which also simplifies
+deployments).
+
+Look for blog posts here and from Google, Solo.io and the Envoy community for
+much more detail about this exciting work!
+
+## Easier to use
+
+We’re always making Istio easier to adopt and use, and this release in
+particular has some cool enhancements. Command line installation of Istio using
+[`istioctl`](/docs/reference/commands/istioctl) is now beta for installation and
+will work for most customers in most use cases. Managing your installation via
+an Operator is still alpha, but we continue to improve it with a new
+[`IstioOperator API`](/docs/reference/config/istio.operator.v1alpha1/).
+
+Speaking of `istioctl`, it has over a dozen improvements -- new items it can
+analyze, better validation rules, and better ability to integrate with CI
+systems (look for examples coming soon!). It is
+now an essential tool for understanding the state of a running Istio system and
+for ensuring that configuration changes are safe. And `istioctl analyze` has
+graduated from the Experimental stage.
+
+We have made numerous enhancements to Istio security to make it easier to use.
+mTLS configuration is simplified and automated with the Beta launch of auto
+mTLS.  We have simplified access control by removing indirection and
+consolidating to a single CRD with the beta launch of authorization policy in
+Istio 1.4.
+
+## More secure
+
+As always, we are working to make Istio more secure with every release. With
+1.5, all security policies including
+[Auto m TLS](/docs/tasks/security/authentication/auto-mtls/),
+[`AuthenticationPolicy`](/docs/reference/config/security/istio.authentication.v1alpha1/)
+(`PeerAuthentication` and `RequestAuthentication`) and authorization are now in
+Beta. SDS is now stable. Authorization now supports Deny semantics to enforce
+mandatory controls that cannot be overridden. We have combined the Node agent
+and the Istio agent into a single binary, which means we no longer require
+configuration of a `PodSecurityPolicy`.
+
+Look for blog posts in the coming days for a deeper dive on Istio security and
+the threats that it helps mitigate.
+
+## Better observability
+
+We continue to invest in making Istio the best way to understand your
+distributed applications. Telemetry v2 now reports metrics for raw TCP
```

Comment:
state that Telemetry v2 (mixerless telemetry) is now the default.

Reply:
done